hsmd/hsmd_wire.csv

```diff
@@ -89,6 +89,17 @@ msgdata,hsmd_setup_channel,channel_type,channel_type,
 # No value returned.
 msgtype,hsmd_setup_channel_reply,131
 
+# Sent to derive the next funding pubkey (for splicing)
+msgtype,hsmd_next_funding_pubkey,34
+msgdata,hsmd_next_funding_pubkey,peerid,node_id,
+msgdata,hsmd_next_funding_pubkey,dbid,u64,
+msgdata,hsmd_next_funding_pubkey,funding_txid,bitcoin_txid,
+msgdata,hsmd_next_funding_pubkey,funding_txout,u32,
```

Comment:
I found this while cherripick the commits, we need it in cln? or it is just an experiment in vls?

Reply:
you will need it once CLN wants to support funding pubkey rotation, but that would be a reasonably big change and is not needed right away (can keep the key unrotated with some loss of privacy).

Reply:
oh nice thanks! so will drop it for now